packages/slate-react/src/utils/dom.ts

```diff
@@ -145,3 +145,32 @@ export const getEditableChild = (
 
   return child
 }
+
+/**
+ * Get a plaintext representation of the content of a node, accounting for block
+ * elements which get a newline appended.
+ *
+ * The domNode must be attached to the DOM.
+ */
+
+export const getPlainText = (domNode: DOMNode) => {
```

Comment:
Moved this into utils, so it could be accessed more easily, and seemed like a reasonable home.